go/base/tests/test_go_start_conversation.py

```diff
@@ -18,8 +18,6 @@ def setUp(self):
         self.command.stderr = StringIO()
 
     def test_sanity_checks(self):
-        self.assertRaisesRegexp(CommandError, 'provide --email-address',
-            self.command.handle, email_address=None, conversation_key=None)
```

Comment:
To think of it another way, this tests the command from the outside. We know that it should complain if we don't give it an email address, but we don't know where the code that checks that lives. If we change this to not use `BaseGoCommand` for some reason, we could break the command's handling of the email address without any tests failing.